aea/cli/create.py

```diff
@@ -95,6 +95,15 @@ def create(click_context, agent_name):
     try:
         path.mkdir(exist_ok=False)
 
+        # set up packages directories.
+        _setup_package_folder(Path(agent_name, "protocols"))
+        _setup_package_folder(Path(agent_name, "connections"))
+        _setup_package_folder(Path(agent_name, "skills"))
+
+        # set up a vendor directory
+        Path(agent_name, "vendor").mkdir(exist_ok=False)
+        Path(agent_name, "vendor", "__init__.py").touch(exist_ok=False)
```

Comment:
these lines have been moved from below, although not needed now. It will be useful when we will check the fingerprint also in the `aea-config.yaml` file.